procedures/katello/branch.md.erb

```diff
@@ -1,7 +1,6 @@
 ## Roles
 
 * Release Owner: <%= owner %>
-* Release Engineer: <%= engineer %>
```

Comment:
A note: the script will still prompt for this. We can leave it out of scope for this PR, but something we should eventually deal with.